f2/src/__tests__/BusinessPage.test.js

```diff
@@ -13,19 +13,20 @@ i18n.load('en', { en })
 i18n.activate('en')
 
 describe('<BusinessPage />', () => {
-    afterEach(cleanup)
+  beforeEach(() => (global.scrollTo = jest.fn()))
```

Comment:
The `window` object that the app uses only exists when running in a browser, not when running in jest. So for the tests to use `window.scrollTo()`, we make the function manually by setting it to a fake jest function. And for some reason we set `global.scrollTo` rather than `window.scrollTo` 🤷‍♂